index.html

```diff
@@ -4,45 +4,240 @@
+          </div>
+          <div class="project">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <div class="languages">
+              <ul>
+                <li class="project-language">html</li>
+                <li></li>
+                <li class="project-language">bootstrap</li>
+                <li></li>
+                <li class="project-language">Ruby</li>
+                <li></li>
+              </ul>
+            </div>
+            <button type="button" class="project-btn">See project</button>
+          </div>
+          <div class="project">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <div class="languages">
+              <ul>
+                <li class="project-language">html</li>
+                <li></li>
+                <li class="project-language">bootstrap</li>
+                <li></li>
+                <li class="project-language">Ruby</li>
+                <li></li>
+              </ul>
+            </div>
+            <button type="button" class="project-btn">See project</button>
+          </div>
+          <div class="project">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <div class="languages">
+              <ul>
+                <li class="project-language">html</li>
+                <li></li>
+                <li class="project-language">bootstrap</li>
+                <li></li>
+                <li class="project-language">Ruby</li>
+                <li></li>
+              </ul>
+            </div>
+            <button type="button" class="project-btn">See project</button>
+          </div>
+          <div class="project">
+            <h2>Profesional Art Printing Data</h2>
+            <p>
+              A daily selection of privately personalized reads no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <div class="languages">
+              <ul>
+                <li class="project-language">html</li>
+                <li></li>
+                <li class="project-language">bootstrap</li>
+                <li></li>
+                <li class="project-language">Ruby</li>
+                <li></li>
+              </ul>
+            </div>
+            <button type="button" class="project-btn">See project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about">
```

Comment:
- [ ] The same case as your `about section`. Would you please double check the link linking of your CSS file  to HTML file.

### Your Design

![image](https://user-images.githubusercontent.com/98453979/185763794-31755c45-9568-4128-b09e-7aca32548519.png)

### Figma design

![image](https://user-images.githubusercontent.com/98453979/185763815-2fa4dd88-87f3-4003-8c31-ce225fd31fce.png)

Reply:
Thank you @frank1738 for the review,let me Implement the requested Changes.